geospaas_harvesting/ingesters.py

```diff
@@ -108,7 +108,7 @@ def _ingest_dataset(self, url, normalized_attributes):
                 platform=platform,
                 instrument=instrument,
                 specs=normalized_attributes.pop('specs', ''))
-
+            geos_para_list = normalized_attributes.pop('geospaas_parameters')
```

Comment:
In the line of my other comment on line 280:
```suggestion
            parameters_list = normalized_attributes.pop('parameters')
```